lib/crypto/c_src/crypto.c

```diff
@@ -90,6 +91,7 @@ static ErlNifFunc nif_funcs[] = {
     {"rand_uniform_nif", 2, rand_uniform_nif, 0},
     {"mod_exp_nif", 4, mod_exp_nif, 0},
     {"do_exor", 2, do_exor, 0},
+    {"pbkdf2_hmac_nif", 5, pbkdf2_hmac_nif, 0},
```

Comment:
I'm actually surprised that such a function is not using timeslices nor it is run in a dirty scheduler. Actually that makes me wonder of all other CPU hungry functions here that aren't run on dirty scheduler either, am I getting something wrong?